### ISSUE TITLE
Claiming Property Tool - Missing Land Claim Papers Step

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#11028]
As of one of my last issues you removed the tutorial step to get land claim papers, that's not what i wanted though. You NEED to have them in the inventory for the land claim stake to work, but you do NOT NEED to select them. What i wanted to achieve is, that the tutorial tells you to put them in your inventory, but does not tell you to select them. Now people probably don't get they need the claim papers and wonder why they can't claim.

TL;DR:
Implement a tutorial step to put the claims into your inventory, but do not ask the user to select it.

**Comment by johnkslg** [comment]
Fixed!
